packages/rds/initial-bootstrap.ts

```diff
@@ -0,0 +1,88 @@
+import { getAuthPassword } from "./auth/secrets";
+import { createConnection, Connection } from "typeorm";
+
+import { CloudFormationCustomResourceEvent, CloudFormationCustomResourceResponse } from "aws-lambda";
+
+type User = {
+  name: string;
+  dbPrivileges: string[];
+};
+
+const DB_NAME = "atat";
+const USERS: User[] = [
+  {
+    name: "atat-read",
```

Comment:
We probably need to name these something like `atat-api-read` so that it's clear that it's not a system-wide shared account. Each "service", however we define it, will need its own account / set of accounts.

Reply:
Sounds good -- used that name.